apps/demo/src/App.tsx

```diff
@@ -92,8 +94,15 @@ export const App = () => {
             alt="Background pattern"
           />
         )}
-        {page === Page.HOME && <Home changePage={(p) => setPage(p)} />}
-        {page === Page.CREATE && <Create changePage={(p) => setPage(p)} />}
+        <Home modal={modal} />
+        <Modal
```

Comment:
WDYT about having the character creation in a modal like this (large instead of full) ? ( I did the alignement quickly just to showcase, it's not perfect )

<img width="1716" alt="Capture d’écran 2024-02-15 à 15 07 20" src="https://github.com/openfun/cunningham/assets/9628870/0ec56162-0f28-49b8-af63-2ceefee92131">